src/server/env.ts

```diff
@@ -80,6 +81,7 @@ const envSpec: EnvSpec = {
     transform: stringToNumber(),
     schema: number().default(86400),
   },
+  METROLINE_CLONE_PROTO: { schema: string() },
```

Comment:
You'll want to use `string()..optional()` just to make sure

Reply:
Shouldn't it be like this? `string().default(ssh)`

Reply:
Oh yeah right, sorry I wasn't paying attention :)